xds/src/test/java/io/grpc/xds/internal/sds/SdsProtocolNegotiatorsTest.java

```diff
@@ -189,11 +194,14 @@ protected void onException(Throwable throwable) {
     // ProtocolNegotiators.ClientTlsHandler.class not accessible, get canonical name
     assertThat(iterator.next().getValue().getClass().getCanonicalName())
         .contains("ProtocolNegotiators.ClientTlsHandler");
+    CommonCertProviderTestUtils.register0();
```

Comment:
As mentioned above, can we define an `unregister` and call it here? It passes the value returned by `register` on line 151.